contracts/optimistic-ethereum/OVM/chain/OVM_StateCommitmentChain.sol

```diff
@@ -244,69 +239,6 @@ contract OVM_StateCommitmentChain is iOVM_StateCommitmentChain, iRingBufferOverw
         return SafeMath.add(timestamp, FRAUD_PROOF_WINDOW) > block.timestamp;
     }
 
-    /**
```

Comment:
We're removing the overwriting logic for now, will be re-enabled later. Not really necessary for minnet.